lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/Constants.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.bridge.client;
+
+import org.apache.reef.client.DriverConfiguration;
+import org.apache.reef.client.DriverServiceConfiguration;
+import org.apache.reef.io.network.naming.NameServerConfiguration;
+import org.apache.reef.javabridge.generic.JobDriver;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.Configurations;
+import org.apache.reef.webserver.HttpHandlerConfiguration;
+import org.apache.reef.webserver.HttpServerReefEventHandler;
+import org.apache.reef.webserver.ReefEventStateManager;
+
+/**
+ * Constant Configuration instances used by the bridge.
+ */
+public final class Constants {
+
+  /**
+   * Contains all bindings of event handlers to the bridge.
+   */
+  public static final Configuration DRIVER_CONFIGURATION = DriverConfiguration.CONF
```

Comment:
For the purpose of the bridge, we always use the *same* Java-side Driver Configurations. Hence, `Constants` seemed appropriate.